### PR TITLE
Quiz No Account

### PR DESCRIPTION
**Pull request by belosanj**

### CODE REVIEW

File: src/screens/noAcctHelperFunctions.tsx [hunk 1]
```diff
@@ -3,6 +3,12 @@ import * as FS from 'expo-file-system';
 const OFFLINE_STATE_URI = `${FS.documentDirectory}/offline_state.txt`;
 
 export interface OfflineContact {
+  id: string;
+  image: {
+    url: string;
+  };
+  url: string;
+  owner: string;
```
Comment (eric-weischedel) [inline src/screens/noAcctHelperFunctions.tsx:11]: Do these fields need to be added to the interface? It seems like for offline contacts, they would have no use.

File: src/screens/QuizScreenNoAcctScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,337 @@
...
+// interface State {
+//     menuVisible: boolean;
+//     searchQuery: string;
+//     contacts: OfflineContact[];
+//     questionResults: QuestionResult[];
+//     QuizTitleListName: string;
+//     CurrentQuizQuestionNumber: number;
+//     loading: boolean;
+//     refreshing: boolean;
+//   }
```
Comment (eric-weischedel) [inline src/screens/QuizScreenNoAcctScreen.tsx:42]: Is there reason to keep this in commented out?

File: src/screens/QuizScreenNoAcctScreen.tsx [hunk 3]
```diff
@@ -0,0 +1,337 @@
...
+    super(props);
+    // this.state = {
+    //     menuVisible: false,
+    //     searchQuery: '',
+    //     //contacts: [],
+    //     questionResults: [],
+    //     QuizTitleListName: '',
+    //     loading: true,
+    //     refreshing: false,
+    //   };
```
Comment (eric-weischedel) [inline src/screens/QuizScreenNoAcctScreen.tsx:57]: Do we need this?

**Review by eric-weischedel — APPROVED** [review]
It works! 🎉 
Left a couple small changes to consider.

File: src/screens/noAcctHelperFunctions.tsx (hunk 1 above)
Comment (belosanj) [reply src/screens/noAcctHelperFunctions.tsx:11]: I was getting a bunch of errors before I added them so I think they do.

File: src/screens/QuizScreenNoAcctScreen.tsx (hunk 2 above)
Comment (belosanj) [reply src/screens/QuizScreenNoAcctScreen.tsx:42]: Nope, I just forgot to remove it. I can do it sometime today

File: src/screens/QuizScreenNoAcctScreen.tsx (hunk 3 above)
Comment (belosanj) [reply src/screens/QuizScreenNoAcctScreen.tsx:57]: No, we don't. I can remove it sometime today after my classes.

**Review by eric-weischedel — APPROVED** [review]

**Pull request merged**